openedx_ltistore/serializers.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
```

Comment:
Remove this dependency to `rest_framework`. Do you think you can use `from django.core import serializers
` do to the job here?

Reply:
I replaced this with `django.forms.models.model_to_dict`